functions/helpers.bash

```diff
@@ -132,3 +132,18 @@ is_xenial() {
 running_in_docker() {
   grep -q 'docker\|lxc' /proc/1/cgroup
 }
+tryUntil() {
```

Comment:
I would like this function to have some unit test so we know that it is working properly.

Reply:
I've created tests in functions/helpers.bats that work from a shell but fail when used with BATS .. it is driving me crazy, please have yourrrself a look.

Reply:
I will look into this in the end of the week. Don't get me wrong I really want this function in the codebase but we need to have firm tests to first. I don't want to introduce more technical debt while trying to iron out old code.

Reply:
'cost me quite some nerves but here you are now.